code/__DEFINES/icon_smoothing.dm

```diff
@@ -192,7 +192,13 @@ DEFINE_BITFIELD(smoothing_junction, list(
 
 #define SMOOTH_GROUP_CLEANABLE_DIRT S_OBJ(68) ///obj/effect/decal/cleanable/dirt
 
-#define SMOOTH_GROUP_GAS_TANK S_OBJ(72)
+#define SMOOTH_GROUP_SPIDER_WEB S_OBJ(69) // /obj/structure/spider/stickyweb
+#define SMOOTH_GROUP_SPIDER_WEB_WALL S_OBJ(70) // /obj/structure/spider/stickyweb/sealed
+#define SMOOTH_GROUP_SPIDER_WEB_WALL_TOUGH S_OBJ(71) // /obj/structure/spider/stickyweb/sealed/thick
+#define SMOOTH_GROUP_SPIDER_WEB_WALL_MIRROR S_OBJ(72) // /obj/structure/spider/stickyweb/sealed/reflector
+#define SMOOTH_GROUP_SPIDER_WEB_ROOF S_OBJ(73) // /obj/structure/spider/passage
+
+#define SMOOTH_GROUP_GAS_TANK S_OBJ(74)
```

Comment:
why isn't this one `69` and everything else added after? is there some special reason for this to be last? I'm not familiar with the logic so it would be nice if we could just make it `S_OBJ(999)` or something (if the code supports it) so we don't have to keep pushing it further and further for future PRs to keep addressing.